### ISSUE TITLE
Version output is sent to STDERR instead of STDOUT

### ISSUE THREAD

**Issue by zouhair** [p-gen/smenu#10]
Using Cygwin:

```
$ smenu -V
Version: 0.9.12-eff1fe6
$ var=$(smenu -V)
$ echo "$var"

$ var=$(smenu -V 2>&1)
$ echo "$var"
Version: 0.9.12-eff1fe6
$ smenu -V | sed -e 's/.* \(.*\)-.*/\1/'
Version: 0.9.12-eff1fe6
$ smenu -V 2>&1 | sed -e 's/.* \(.*\)-.*/\1/'
0.9.12
$
```

I don't know if it is on purpose or not, so here it goes.

**Comment by p-gen** [comment]
Hi, fixed by commit 95c9eb5

**Comment by zouhair** [comment]
Thanks
